README.md

```diff
@@ -101,7 +101,7 @@ Each device must have a unique device ID which adhere to the [ID Format](#id-for
 #### Device Attributes
 
 * `homie` / `device ID` / **`$device-attribute`**:
-A device attribute MUST be one of these:
+The set of device attributes MUST be sent immediately after connection to MQTT broker is established and MUST be one of these:
```

Comment:
"MUST be sent" is a onesided view from the eyes of a IoT developer. We should also cover the Controller view.